src/output-json-alert.c

```diff
@@ -243,6 +243,10 @@ void AlertJsonHeader(const Packet *p, const PacketAlert *pa, json_t *js)
         }
     }
 
+    if (p->flow) {
+        json_object_set_new(js, "app_proto",
```

Comment:
What I meant on the doc side is to have certain options appear *only* in the docs, so that the default yaml we ship isn't as verbose. So in this example, if we add a 'flow' switch to trigger flow logging, then don't add it to the default yaml, but do add docs for it. So ppl can disable it, but by default it's enabled and hidden from yaml. Make sense?